### PR TITLE
[Doc] Add Chinese doc for tutorial02_customized_dataset_md

### PR DESCRIPTION
**Pull request by MengzhangLI**
Add Chinese doc for tutorial02_customized_dataset_md

### CODE REVIEW

File: docs_zh-CN/tutorials/customize_datasets.md [hunk 1]
```diff
@@ -1 +1,173 @@
...
+`data/my_dataset/ann_dir/train/xxx{seg_map_suffix}`,
+`data/my_dataset/ann_dir/train/zzz{seg_map_suffix}` 将被加载。
+
+注意：注释是跟图像同样的形状 (H, W)，其中的像素值的范围是 `[0, num_classes - 1]`。
+您也可以使用 [pillow](https://pillow.readthedocs.io/en/stable/handbook/concepts.html#palette) 的 `'P'` 模式去创建您的注释，包括了注释图像的颜色。
+
+## 通过混合数据去定制数据集
+
+MMSegmentation 同样支持混合数据集去训练。
+当前它支持连接 (concat) 和 重复 (repeat) 数据集。
```
Comment (xvjiarui) [inline docs_zh-CN/tutorials/customize_datasets.md:52]:
```suggestion
当前它支持串联 (concat) 和 重复 (repeat) 数据集。
```

File: docs_zh-CN/tutorials/customize_datasets.md [hunk 2]
```diff
@@ -1 +1,173 @@
...
+        ```python
+        dataset_A_train = dict(
+            type='Dataset_A',
+            img_dir = 'img_dir',
+            ann_dir = ['anno_dir_1', 'anno_dir_2'],
+            pipeline=train_pipeline
+        )
+        ```
+
+    2. 您也可以去连接两个 `split` 文本。
```
Comment (xvjiarui) [inline docs_zh-CN/tutorials/customize_datasets.md:89]:
```suggestion
    2. 您也可以去串联两个 `split` 文本。
```

File: docs_zh-CN/tutorials/customize_datasets.md [hunk 3]
```diff
@@ -1 +1,173 @@
...
+        train = [
+            dataset_A_train,
+            dataset_B_train
+        ],
+        val = dataset_A_val,
+        test = dataset_A_test
+        )
+    ```
+
+一个更复杂的例子如下：分别重复 `Dataset_A` 和 `Dataset_B` N 次和 M 次，然后再去连接重复后的数据集。
```
Comment (xvjiarui) [inline docs_zh-CN/tutorials/customize_datasets.md:133]:
```suggestion
一个更复杂的例子如下：分别重复 `Dataset_A` 和 `Dataset_B` N 次和 M 次，然后再去串联重复后的数据集。
```

**Review by Junjun2016 — APPROVED** [review]

**Comment by Junjun2016** [comment]
Hi @MengzhangLI 
Please fix the conflicts.

**Pull request merged**